HomeWork/src/main/java/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,56 @@
+package coffee.order;
+
+import java.util.*;
+
+public class CoffeeOrderBoard {
+    private Integer last;
+
+    private LinkedList<Order> listOrder = new LinkedList<>();
+
+    public LinkedList<Order> getListOrder() {
+        return listOrder;
+    }
+
+    public boolean add(Order order) {
+        Objects.requireNonNull(order);
+        if (listOrder.isEmpty()) {
+            listOrder.add(order);
+            order.setOrderNumber(last = listOrder.size());
```

Comment:
The order number should be calculated just by incrementing it. So, if you create an order, do `last++`, which is the last order number you're looking for.